Sources/web3swift/Browser/Bridge.swift

```diff
@@ -8,122 +8,122 @@
 
 import WebKit
 
-/// Bridge for WKWebView and JavaScript
+// Bridge for WKWebView and JavaScript
```

Comment:
I wouldn't change documentation comments to simple comments as it disables "Quick Help" and won't be picked up by documentation page generators like jazzy from Realm.

Before:
<img width="265" alt="Screenshot 2022-03-22 at 14 49 21" src="https://user-images.githubusercontent.com/36865532/159485719-0577fab6-9866-4331-9d19-485a9c2bdaed.png">
<img width="516" alt="Screenshot 2022-03-22 at 14 50 32" src="https://user-images.githubusercontent.com/36865532/159485726-67fda579-8221-4a93-8eac-d6b7cfbe7a33.png">

After:
<img width="270" alt="Screenshot 2022-03-22 at 14 49 32" src="https://user-images.githubusercontent.com/36865532/159485814-c8c5cfaf-8547-4d4f-b9e4-0ff8dd946c88.png">
<img width="505" alt="Screenshot 2022-03-22 at 14 52 12" src="https://user-images.githubusercontent.com/36865532/159485819-ba6908c0-01cc-483d-b5b2-14c5ce450849.png">